scripts/bundle-rhmi-operators.sh

```diff
@@ -126,18 +130,10 @@ push_index() {
   docker push $INDEX_IMAGE
 }
 
-# creates catalog source on the cluster
-create_catalog_source() {
```

Comment:
why remove this , is it no longer valid ?

Reply:
It actually  only works as expected with a single image and gets too messy for what's its giving when it comes to more index images

Reply:
Actually found a reasonable solution. 
Added it back in but only the latest version is picked up, also, only if it has no replaces field. This seems to me like only reasonable solution. Let me know if you have any other ideas